fe/fe-core/src/main/java/com/starrocks/storagevolume/StorageVolume.java

```diff
@@ -107,6 +110,25 @@ public StorageVolume(StorageVolume sv) {
         this.enabled = sv.enabled;
         this.cloudConfiguration = CloudConfigurationFactory.buildCloudConfigurationForStorage(sv.params, true);
         this.params = new HashMap<>(sv.params);
+        validateStorageVolumeConstraints();
+    }
+
+    private void validateStorageVolumeConstraints() throws DdlException {
```

Comment:
seems not necessary?

Reply:
subpath is not allowed when storage volume is enabled with partitioned prefix, otherwise the meaning of subpath is confusing.